templates/block/block--menu-block--news-topics.html.twig

```diff
@@ -42,15 +42,18 @@
 {% set menu_class = 'news-topics__collapsable-menu' %}
 {% set heading_id = (configuration.id ~ '-menu')|clean_id %}
 {% set attributes = attributes.setAttribute('id', heading_id) %}
+{% set button_label = ('button-label')|clean_id %}
+{% set attributes = attributes.setAttribute('aria-labelledby', button_label) %}
+{% set button_id = ('button-label')|clean_id %}
```

Comment:
Because you already have one button-label id this will generate something that looks like `button-label-0`. I think what you want here is to remove the `button_id` variable on line 47 and to change line 56 to use the `button_label` variable. That way both the id and the labelledby attributes use the name string.

Reply:
Oh I see. On it! Thank you. I'll fix for all of them.